CalibTracker/SiPixelLorentzAngle/src/SiPixelLorentzAnglePCLWorker.cc

```diff
@@ -536,6 +536,22 @@ void SiPixelLorentzAnglePCLWorker::analyze(edm::Event const& iEvent, edm::EventS
                 iHists.h_drift_depth_adc2_.at(i_index)->Fill(drift, depth, pixinfo_.adc[j] * pixinfo_.adc[j]);
                 iHists.h_drift_depth_noadc_.at(i_index)->Fill(drift, depth, 1.);
                 iHists.h_bySectOccupancy_->Fill(i_index - 1);  // histogram starts at 0
+
+                if ((module_ == 3 || module_ == 5) && (layer_ == 3 || layer_ == 4)) {
```

Comment:
@wweiphy I think this grouping will work only for the Phase-1 detector, as the Phase-2 one has a different structure with 9 modules in a BPix ladder:

![image](https://user-images.githubusercontent.com/5082376/155095963-abc44eba-347a-4430-934f-f4fef804fc2e.png)

should this be gated for the phase-1 case?